src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+    add(number) {
+      calculator.result += number;
+    },
+    subtract(number) {
+      calculator.result -= number;
+    },
+    multiply(number) {
+      calculator.result *= number;
+    },
+    divide(number) {
+      calculator.result /= number;
```

Comment:
do not interact with result in this methods return result of calculation and assign it in operate method